main/fakeroot/APKBUILD

```diff
@@ -1,34 +1,32 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=fakeroot
-pkgver=1.22
+pkgver=1.23
 pkgrel=0
 pkgdesc="Gives a fake root environment, useful for building packages as a non-privileged user"
 arch="all"
-license='GPL'
-url="http://fakeroot.alioth.debian.org/"
+license="GPL-3.0-or-later"
+url="https://packages.debian.org/fakeroot"
 depends=
 options="!checkroot"
 checkdepends="bash"
-makedepends_build="libtool autoconf automake po4a"
-makedepends_host="libcap-dev acl-dev linux-headers"
-makedepends="$makedepends_build $makedepends_host"
+makedepends="acl-dev autoconf automake libcap-dev libtool linux-headers po4a"
```

Comment:
Why did you remove `makedepends_build` and `makedepends_host`? These are needed for cross-compilation.

Reply:
@jirutka Apologies, i didn't know about that.
Reverted.